src/client/src/apps/utils/openfin-utils.ts

```diff
@@ -47,14 +47,15 @@ export async function createAndRunOpenFinApplication({
   name,
   url,
   uuid,
-  windowOptions
+  windowOptions,
 }: ApplicationConfig): Promise<Application> {
   const appOptions: fin.ApplicationOption = {
     name,
     url,
     uuid: uuid || name,
     nonPersistent: true,
-    mainWindowOptions: windowOptions
+    mainWindowOptions: windowOptions,
+    fdc3Api: true,
```

Comment:
Is it possible to add this option per application in applicationConfigurations.tsx instead, so that it's more explicit?

It also means it wouldn't be automatically enabled for all openfin apps, we probably want to do it on an opt-in basis.